examples/svelte/localized-sveltekit/src/hooks.server.js

```diff
@@ -0,0 +1,48 @@
+import { getLocaleFromPathname } from './utils';
+const defaultLocale = 'en';
+// Match this with the locales defined in your builder space
+const supportedLocales = ['en', 'fr', 'de'];
+const routeRegex = new RegExp(/^\/[^.]*([?#].*)?$/);
```

Comment:
Would be great to explain this regex with a comment, and a link to a regex fiddle like https://regexr.com.

To an outsider (like me), I have no idea what this is looking up, it's somewhat cryptic.